go-controller/go.mod

```diff
@@ -25,13 +25,11 @@ require (
 	github.com/onsi/gomega v1.8.1
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.2.1
-	github.com/satori/go.uuid v0.0.0-20181028125025-b2ce2384e17b // indirect
-	github.com/spf13/afero v1.2.2
-	github.com/stretchr/testify v1.4.0
+	github.com/spf13/afero v1.4.1
+	github.com/stretchr/testify v1.5.1
 	github.com/urfave/cli/v2 v2.2.0
-	github.com/vishvananda/netlink v0.0.0-20200625175047-bca67dfc8220
+	github.com/vishvananda/netlink v1.1.0
```

Comment:
why do you change this if you want to use `v0.0.0-20200625175047-bca67dfc8220` ?

Reply:
ahh sorry, it is explained in the commit message
```
add explicit replace directive in go.mod to keep current
version of netlink as sriovnet will downgrade it
```

we should update in the sriovnet project then

Reply:
netlink doesnt really have a release cadence, which is an issue.

sriovnet is based on a released version of netlink, which id like to keep. 
once a newer version of netlink is released i think both projects should update to an official release version.

i added this as conflicting versions messes up vendoring support, hence i keep the newer version.

go mod seems to always prefers release version >=v1 over v0.x.x

Reply:
that make sense, thanks for the clarification